build.gradle

```diff
@@ -24,7 +23,6 @@ buildscript {
         classpath 'net.ltgt.gradle:gradle-apt-plugin:0.5' // for non-android projects
```

Comment:
I don't believe we are using this apt either.  Want to give it a yank and see if it still builds?